include/api.php

```diff
@@ -114,12 +114,12 @@ function pll_default_language( $field = 'slug' ) {
  *
  * @param int                 $post_id Post ID.
  * @param PLL_Language|string $lang    Optional language (object or slug), defaults to the current language.
- * @return int|false The translation post ID if exists, otherwise the passed ID. False if the passed object has no language or if the language doesn't exist.
+ * @return int|false The translation post ID if exists, 0 if not translated or if the post has no language. False if the language doesn't exist.
  *
  * @phpstan-return int<0, max>|false
  */
-function pll_get_post( $post_id, $lang = '' ) {
-	$lang = $lang ? $lang : pll_current_language();
+function pll_get_post( $post_id, $lang = null ) {
```

Comment:
Historically this function was meant to accept a language code. And the default value `pll_current_language()` returns a string, so I would have kept the empty string as param,
Not very important though.

Reply:
Reverted in 6ff20f1d, also use empty string in `pll_get_term()` for constistency.